portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,7 +75,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
               (String) entity.getProperty("content"), 
               (long) entity.getKey().getId(),
               (long) entity.getProperty("likeCount"),
-              (String) entity.getProperty("email"))
+              (String) entity.getProperty("email"),
+              (float)(double) entity.getProperty("sentimentScore"))
```

Comment:
Could you explain this double conversion?

Reply:
It is because Datastore stores floats as doubles, widening the original float. Just added a comment for this line!